### DISCUSSION TITLE
Mojo specific test framework?

### DISCUSSION THREAD

**Discussion by masters3d** [modular/modular#2505] [Q&A] [answered]
Usually when I explore a new language, I like to start with writing tests. I see there is a plug in for pytest https://github.com/guidorice/mojo-pytest

Is anybody aware of a Mojo specific testing library? Im interested in exploring only mojo specific features.

**Comment by Benny-Nottonson** [comment, marked as answer]
There are no public mojo testing frameworks at the moment, although there is https://docs.modular.com/mojo/stdlib/testing/testing, as well as `contrained[condition, msg]`.
